plugins/english/lnmtl.ts

```diff
@@ -86,52 +95,68 @@ class LnMTLPlugin implements Plugin.PluginBase {
+        return novel;
+    };
+
+    async parsePage(novelPath: string, page: string): Promise<Plugin.SourcePage> {
+        const result = await fetchApi(this.site + novelPath);
+        const body = await result.text();
+
+        const loadedCheerio = parseHTML(body);
+
+        const volumes = JSON.parse(
             loadedCheerio("main")
                 .next()
                 .html()
                 ?.match(/lnmtl.volumes = \[(.*?)\]/)![0]
                 ?.replace("lnmtl.volumes = ", "") || ""
-        );
+        )[+page-1];
 
         const chapter: Plugin.ChapterItem[] = [];
+        interface ChapterEntry {
+            number: number;
+            title: string;
+            slug: string;
+            created_at: string;
+        }
 
-        volumes = volumes.map((volume: { id: number; }) => volume.id);
-
-        for (const volume of volumes) {
-            const volumeData = await fetchApi(
-                `${this.site}chapter?page=1&volumeId=${volume}`
+        await this.sleep(1000);
+        const volumeData = await fetchApi(
+            `${this.site}chapter?volumeId=${volumes.id}`
+        );
+        const volumePage = await volumeData.json();
+        const firstPage = volumePage.data.map(
+            (chapter: ChapterEntry) => ({
+                page: `Volume ${volumes.number} - ${volumes.title}`,
+                name: `#${chapter.number} - ${chapter.title}`,
+                path: `chapter/${chapter.slug}`,
+                releaseTime: new Date (chapter.created_at).toISOString(), //converts time obtained to UTC +0, TODO: Make it not convert
+        }))
+        chapter.push(...firstPage);
+
+        for (let i = 2; i <= volumePage.last_page; i++) {
+            await this.sleep(1000);
+            const chapterData = await fetchApi(
+                `${this.site}chapter?page=${i}&volumeId=${volumes.id}`
             );
-            const volumePage = await volumeData.json();
-
-            // volumeData = volumeData.data.map((volume) => volume.slug);
-
-            for (let i = 1; i <= volumePage.last_page; i++) {
-                const chapterData = await fetchApi(
-                    `${this.site}chapter?page=${i}&volumeId=${volume}`
-                );
-                const chapterInfo = await chapterData.json();
+            const chapterInfo = await chapterData.json();
 
-                const chapterDetails = chapterInfo.data.map(
-                (chapter: { number: number; title: string; slug: string; created_at: string; }) => ({
-                    name: `#${chapter.number} ${chapter.title}`,
-                    url: `${this.site}chapter/${chapter.slug}`,
-                    releaseTime: new Date (chapter.created_at).toISOString(), //converts time obtained to UTC +0, TODO: Make it not convert
-                }));
+            const chapterDetails = chapterInfo.data.map(
+            (chapter: ChapterEntry) => ({
+                page: `Volume ${volumes.number} - ${volumes.title}`,
```

Comment:
```suggestion
```
If the novel already had totalPages, adding `page` property for chapter makes no sense